cmake/Tools/Platform/Android/android_deployment.py

```diff
@@ -129,14 +129,14 @@ def __init__(self, dev_root, build_dir, configuration, android_device_filter, cl
     @staticmethod
     def read_android_settings(dev_root, game_name):
         """
-        Read and parse the project.json file into a dictionary to process the specific attributes needed for the manifest template
+        Read and parse the android_project.json file into a dictionary to process the specific attributes needed for the manifest template
 
         :param dev_root:    The dev root we are working from
         :param game_name:   Name of the game under the dev root
         :return: The android settings for the game project if any
         """
         game_folder = dev_root / game_name
-        game_folder_project_properties_path = game_folder / 'project.json'
+        game_folder_project_properties_path = game_folder / 'Platform' / 'Android' / 'android_project.json'
```

Comment:
This should try to read from both files, with the project.json as a fallback.

Reply:
Agreed. They should both be read, the project.json first then merged with <platform>_project.json, for derrics reasons and, though we have no example of doing it, the platform specific could override if it had to.

Reply:
I thought of that but there are few reasons not to do this:
- There are other places in the code where it's checking android_project.json, not project.json, these are the only 2 places remaining. So to support both json files the changes will have to be larger.
- Android cmake tools are difficult to grasp, I rather keep it simple and use the right json file while stabilizing the platform.
- Android platform is still not stable yet, it needs a lot of love, I think we can afford to do this types of changes until it's ready for prime development time.

Reply:
I don't mind the change of android_settings being sourced only from the android_project.json file. It is more "pure" in my opinion and stops users from putting platform specific settings in the project.json.
However if we make the change, it is impactful and would notification to O3DE users.

An impactful change notification would need to be sent in [Discord #impactful-changes](https://discord.com/channels/805939474655346758/852574203437121586) channel.

In addition we should update the [Android Support for Open 3D Engine](https://www.o3de.org/docs/user-guide/platforms/android/#setup-requirements) page to mention that the android_project.json file is where the android_settings for APK naming and orientation are surfaced from.

Reply:
Yes. And reading from android_project.json, since it would be merged last, is guaranteed to have the correct platform specific item.